snapshots.nix

```diff
@@ -19,7 +19,8 @@ let
       ++ [(hackage: ghc-boot-packages.${(pkg-def hackage).compiler.nix-name})];
     modules = [
       { reinstallableLibGhc = true; } # Allow ghc library to be installed for packages that need it
-      { packages.Cabal.patches = [ ./overlays/patches/Cabal/fix-data-dir.patch ]; }
+      # travis: Do we still need this?
+      #{ packages.Cabal.patches = [ ./overlays/patches/Cabal/fix-data-dir.patch ]; }
```

Comment:
See https://github.com/haskell/cabal/issues/5862 for the back story, So this will apply only to Cabal < 3.0.0.0

Reply:
Ok, I'll just case on the snapshot major version, then.